PGM/src/main/java/tc/oc/pgm/polls/commands/PollSubCommands.java

```diff
@@ -0,0 +1,183 @@
+        if(args.getSuggestionContext() != null) {
+            return CommandUtils.completeMapName(mapName);
+        }
+
+        if (!Config.Poll.enabled()) {
+            throw new TranslatableCommandException("poll.disabled");
+        }
+
+        if (restartManager.isRestartRequested()) {
+            throw new TranslatableCommandException("poll.map.restarting");
+        }
+        if (PGM.getMatchManager().hasMapSet()) {
+            throw new TranslatableCommandException("poll.map.alreadyset");
+        }
+        if (sender instanceof Player) {
+            Player player = (Player) sender;
+            if (TokenUtil.getUser(player).maptokens() < 1) {
+                throw new TranslatableCommandException("tokens.map.fail");
+            }
+        }
+
+        PGMMap nextMap = CommandUtils.getMap(args.getJoinedStrings(0), sender);
+
+        if (pollBlacklist.isBlacklisted(nextMap) && !sender.hasPermission("poll.next.override")) {
+            throw new TranslatableCommandException("poll.map.notallowed");
+        }
+
+        if (PGM.get().getServer().getOnlinePlayers().size() * 4 / 5 > nextMap.getDocument().max_players() && !sender.hasPermission("poll.next.override")) {
+            throw new TranslatableCommandException("poll.map.toomanyplayers");
+        }
+
+        pollManager.startPoll(pollMapFactory.create(sender, nextMap));
+        return null;
+    }
+
+    @Command(
+            aliases = {"mutation", "mt"},
+            desc = "Start a poll to set a mutation",
+            usage = "[mutation name]",
+            min = 1,
+            max = -1
+    )
+    @CommandPermissions("poll.mutation")
+    public List<String> pollMutation(CommandContext args, CommandSender sender) throws CommandException {
+
+        if (args.getSuggestionContext() != null) {
+            Collection<Mutation> availableMutations = Sets.newHashSet(Mutation.values());
+            availableMutations.removeAll(mutationQueue.mutations());
+            return StringUtils.complete(args.getSuggestionContext().getPrefix(), availableMutations.stream().map(mutation -> mutation.name().toLowerCase()));
+        }
+
+        if (!Config.Poll.enabled()) {
+            throw new TranslatableCommandException("poll.disabled");
+        }
+
+        if (sender instanceof Player) {
+            Player player = (Player) sender;
+            if (TokenUtil.getUser(player).mutationtokens() < 1) {
+                throw new TranslatableCommandException("tokens.mutation.fail");
+            }
+        }
+
+        String mutationString = args.getString(0);
+
+        Mutation mutation = StringUtils.bestFuzzyMatch(mutationString, Sets.newHashSet(Mutation.values()), 0.9);
+        if(mutation == null) {
+            throw new TranslatableCommandException("command.mutation.error.find", mutationString);
+        } else if(MutationCommands.getInstance().getMutationQueue().mutations().contains(mutation)) {
```

Comment:
Avoid statically referencing other injected commands like `MutationsCommands`. All you need is the `MutationQueue` object, which you already have as a local variable in `mutationQueue`